### ISSUE TITLE
MathQuill leading to unwanted input fields when outputformat=ptx

### ISSUE THREAD

**Issue by Alex-Jordan** [openwebwork/webwork2#1724]
I'm starting testing WW 2.17 with PreTeXt. I'll start posting issues I find that I may need help with.

If the host course is using MathQuill, then the html2xml response with `displayMode=PTX` and `outputformat=ptx` is returning with an HTML `input` element. For example, look at the source here:

https://webwork-dev.aimath.org/webwork2/html2xml?&amp;problemSeed=1&amp;answersSubmitted=0&amp;sourceFilePath=Library/PCC/BasicAlgebra/Geometry/CylinderVolume10.pg&amp;displayMode=PTX&amp;courseID=anonymous&amp;userID=anonymous&amp;course_password=anonymous&amp;outputformat=ptx

Now, I've turned off MathQuill for this course so you won't see the issue. But if I turn it on, then two `input` elements precede the `answerhashes` element:

```
<input type=hidden name="MaThQuIlL_AnSwEr0001" id="MaThQuIlL_AnSwEr0001" value="" ><input type=hidden name="MaThQuIlL_AnSwEr0002" id="MaThQuIlL_AnSwEr0002" value="" >
```

Can you point me to where this comes from? I can think about the right way to clear it from `outputformat=ptx`.

**Comment by drgrice1** [comment]
The mathquill inputs are added in the ENDDOCUMENT subroutine in pg/macros/PG.pl (starting at line 424 in that file).  The inputs are only added if the display mode is HTML.  The problem it seems is that when the display mode is PTX, then the fallback is HTML.  Since the PTX mode is not specified in ENDOCUMENT (lines 478-479), it falls back to HTML mode and displays the inputs.

**Comment by drgrice1** [comment]
This is fixed, right @Alex-Jordan?

**Comment by drgrice1** [comment]
Fixed by openwebwork/pg#695
